src/pages/ReimbursementAccount/ValidationStep.js

```diff
@@ -188,28 +189,32 @@ class ValidationStep extends React.Component {
                     <View style={[styles.flex1]}>
                         <Section
                             title={this.props.translate('workspace.bankAccount.letsFinishInChat')}
-                            icon={Illustrations.ConciergeBlue}
+                            icon={Illustrations.ConciergeBubble}
                         >
                             <Text>
                                 {this.props.translate('validationStep.letsChatText')}
                             </Text>
+                            <Button
+                                text={this.props.translate('validationStep.letsChatCTA')}
+                                onPress={Report.navigateToConciergeChat}
+                                icon={Expensicons.ChatBubble}
+                                style={[styles.mt4]}
+                                iconStyles={[styles.buttonCTAIcon]}
+                                shouldShowRightIcon
+                                large
+                                success
+                            />
+                            <View style={[styles.mv3]}>
+                                <MenuItem
+                                    title={this.props.translate('workspace.bankAccount.noLetsStartOver')}
+                                    icon={Expensicons.RotateLeft}
+                                    onPress={BankAccounts.requestResetFreePlanBankAccount}
+                                    shouldShowRightIcon
+                                    iconFill={themeColors.success}
+                                    wrapperStyle={[styles.cardMenuItem]}
+                                />
+                            </View>
```

Comment:
```suggestion
                            <MenuItem
                                title={this.props.translate('workspace.bankAccount.noLetsStartOver')}
                                icon={Expensicons.RotateLeft}
                                onPress={BankAccounts.requestResetFreePlanBankAccount}
                                shouldShowRightIcon
                                iconFill={themeColors.success}
                                wrapperStyle={[styles.cardMenuItem, styles.mv3]}
                            />
```
I think the wrapping `View` is not necessary, we can pass the style in `wrapperStyle` directly to the `MenuItem`

Reply:
Same as above, this makes sense to me!